readme.md

```diff
@@ -93,10 +93,16 @@ new PureFullPage();
 When instantiating pureFullPage, an object is accepted as a parameter, which can control whether to display the right navigation (It often does not need the right navigation on mobile) and custom page animation. The sample code is as follows:
 
 ```js
-new PureFullPage({
+const pureFullPage = new PureFullPage({
   isShowNav: true,
   definePages: addAnimation,
 });
+
+pureFullPage.init();
```

Comment:
We needn't call `init` method by ourself because when we initialize `pureFullPage`  we already [called](https://github.com/xiaogliu/pure-full-page/blob/master/src/js/pureFullPage.js#L33) `init` method implicitly.

我们没必要调用 `init` 方法，因为我们在初始化 `pureFullPage` 时已经[隐式调用](https://github.com/xiaogliu/pure-full-page/blob/master/src/js/pureFullPage.js#L33)了。